ci-operator/step-registry/telcov10n/functional/hcp-cnf-nrop/tests/telcov10n-functional-hcp-cnf-nrop-tests-commands.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ NROP Deployment setup command ************"
+# Fix user IDs in a container
+~/fix_uid.sh
+
+date +%s > "${SHARED_DIR}"/start_time
+
+#Environment variables required
+SSH_PKEY_PATH=/var/run/ci-key/cikey
+SSH_PKEY=~/key
+cp $SSH_PKEY_PATH $SSH_PKEY
+chmod 600 $SSH_PKEY
+BASTION_IP="$(cat /var/run/bastion-ip/bastionip)"
+BASTION_USER="$(cat /var/run/bastion-user/bastionuser)"
+COMMON_SSH_ARGS="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o ServerAliveInterval=30"
+T5CI_HCP_MGMT_KUBECONFIG="${T5CI_HCP_MGMT_KUBECONFIG:=${SHARED_DIR}/mgmt-kubeconfig}"
+T5CI_HCP_HOSTED_KUBECONFIG="${T5CI_HCP_HOSTED_KUBECONFIG:=${SHARED_DIR}/kubeconfig}"
+source "${SHARED_DIR}"/main.env
+
+ls -ltR ${SHARED_DIR}
+cat ${SHARED_DIR}/inventory
+
+# Check if Inventory file exists
+if [[ -f "${SHARED_DIR}"/inventory ]]; then
```

Comment:
Sorry my real bad  can you change this to `if ! 1[[ -f "${SHARED_DIR}"/inventory ]]; then`